### DISCUSSION TITLE
copilot didn't work

### DISCUSSION THREAD

**Discussion by sodaball** [community/community#122370] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

My account was verified as a student a year ago and Copilot was used for a year, but a few days ago my Copilot stopped working, why is that?

As shown in the screenshot below：

![img](https://github.com/community/community/assets/41469160/08fcbc22-20ec-4dc0-af79-f17842498ca9)

in vscode：

![img2](https://github.com/community/community/assets/41469160/b47abe2f-3345-4a76-a6bf-f29ff8e87579)

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact

**Reply by sodaball** [reply]
ok, thanks

**Comment by soniasebastian** [comment]
My copilot accounts is not working and asking to contact support
